app/hotels/src/filter/FilterStripe.js

```diff
@@ -17,26 +18,36 @@ const styles = {
   },
 };
 
-export default class FilterStripe extends React.Component<{||}> {
-  handleChange = () => {
-    // TODO add onChange prop, pass changed values via callback
+type Props = {|
+  onChange: FilterParametersType => void,
+  filter?: FilterParametersType,
+|};
+
+export default class FilterStripe extends React.Component<Props> {
+  handleStarsChange = () => {
+    // TODO stars filter handler
   };
 
+  handlePriceChange = (filter: FilterParametersType) =>
+    this.props.onChange(filter);
+
   render = () => {
+    const { filter } = this.props;
+    const minPrice = (filter && filter.minPrice) || MIN_PRICE;
```

Comment:
From my perspective, this should be still moved to PriceFilter itself and you don't need to import max/min price as well.